config/config.md

```diff
@@ -130,7 +130,9 @@ The following config option are provided by the OpenHIM. All of these options ha
   },
   "reports": {
     // To enable daily and weekly reporting
-    "enableReports": true
+    "enableReports": true,
+    //the default timezone offset
+    "utcOffset": +200
```

Comment:
@bradsawadye 
Is +200 correct?
Should it perhaps be "+0200" as it is used in the json config file. It may mislead the user